CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- The `pyodide xbuildenv search` command now accepts a `--json` flag to output the
+  search results in JSON format that is machine-readable.
+  [#28](https://github.com/pyodide/pyodide-build/pull/28)
```

Comment:
Let's make an [`Added` Section](https://keepachangelog.com/en/1.0.0/#how) and put it in that section.

Reply:
Makes sense, addressed in f8f639697f7d873c57dd15f32304d2ace974d0dd

Reply:
Makes sense, addressed in f8f639697f7d873c57dd15f32304d2ace974d0dd.